en/lesson-08.md

```diff
@@ -158,181 +154,217 @@ by using
 ```
 <!-- {% endraw %} -->
 
-(We'll see `\hspace` [again shortly](lesson-11); you might guess that it adds a
-horizontal space.)
-
-In addition to all the aforementioned preamble content, we can use a few other
-things as well
-
-<!-- don't line wrap this table, markdown seems to not support this -->
+## Adding rules
 
-type | description
----  | :--
-`*{num}{string}` | repeats `string` for `num` times in the preamble. With this you can define multiple identical columns.
-`>{decl}` | this will put `decl` before the contents of every cell in the following column (this is useful, e.g., to set a different font for this column)
-`<{decl}` | this will put `decl` after the contents of each cell in the previous column
+A word of advice prior to introducing lines; lines should be used really
+sparsely in tables, and normally vertical ones look unprofessional. In fact,
+for professional tables you shouldn't use any of the standard lines; instead you
+should get familiar with the facilities of the `booktabs` package, which is why
+it is covered here first &ndash; for the sake of completeness I'll show you the
+standard lines later on in the more-info page.
```

Comment:
If the idea is showing best practices, is it worth showing standard rules at all? (I can see reasons to answer “yes”; I'm just raising the question)

Reply:
Another issue is the first person usage.

Reply:
I thought they should be covered just because of the vertical rules. We can throw them out and stick to "don't use vertical rules, ever", but I'm not sure whether this would be the right thing to do.

Reply:
People do want to make 'grids' for good reasons: I think we can but standard lines in 'more'.